### PR TITLE
Wharton events

### PR DESCRIPTION
**Pull request by adamdom**
Added functionality for Wharton events. Testing now included.

### CODE REVIEW

**Comment by adamdom** [comment]
@adelq Brandon and I are not sure why it is failing, but the Travis output shows that it has nothing to do with the new code. Any ideas?

**Comment by adelq** [comment]
I believe these courses are unavailable during the summer, causing the tests to fail. Tests should be fixed, but not in this pull request.

File: README.rst [hunk 1]
```diff
@@ -19,7 +19,7 @@ Getting an API key
 To use these libraries, you must first obtain an API token and password,
 which can be done here_. There are separate API tokens and password for each of dining, registrar, news/events/maps, transit, and directory.
 
-There is also a Laundry API, Study Spaces API, and Calendar, which don't need a key.
+There is also a Laundry API, Study Spaces API, and Calendar (wtih support for Wharton events), which don't need a key.
```
Comment (adelq) [inline README.rst]: Typo

Comment (adamdom) [inline README.rst]: Rip.

Comment (adelq) [inline README.rst]: Still need to fix this typo lol

Comment (adamdom) [inline README.rst]: lol yea doing it now

File: tests/wharton_calendar.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
+from nose.tools import ok_
+from penn import whartonCalendar
+import datetime
+
+class TestCalendar():
+
+    def setUp(self):
+        self.calendar = Calendar()
```
Comment (adelq) [inline tests/wharton_calendar.py]: Not needed, you're not using the Calendar here at all

File: tests/wharton_calendar.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+            end = datetime.datetime.strptime(event['endTime'], '%I:%M %p').date()
+            ok_((end - start).total_seconds() >= 0)
+
+    def test_chrono(self):
+        l = self.whartonCalendar.pull_wharton_events_on_date('2015-09-16')
+        ok_(len(l) > 0)        
+        for i, event in enumerate(l[:-1]):            
+            start = datetime.datetime.strptime(event['startTime'], '%Y-%m-%d').date()
+            nextstart = datetime.datetime.strptime(l[i]['startTime'], '%Y-%m-%d').date()
+            ok_((nextstart - start).total_seconds() >= 0)
```
Comment (adelq) [inline tests/wharton_calendar.py]: I feel the test should be slightly more specific than 0 length...

**Comment by adelq** [comment]
This is currently completely uncached, which is generally a bad idea when scraping a 3rd party page

**Comment by joshdoman** [comment]
No longer functional after Spike was deprecated

**Pull request closed without merging**